docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -36,122 +42,179 @@ Available devices:
+
+    .. doxygensnippet:: docs/snippets/gpu/compile_model.cpp
+        :language: cpp
+        :fragment: [compile_model_gpu_with_id]
+
+.. tab:: Running on specific tile
+
+    .. doxygensnippet:: docs/snippets/gpu/compile_model.cpp
+        :language: cpp
+        :fragment: [compile_model_gpu_with_id_and_tile]
+
+@endsphinxdirective
+
+## Supported inference data types
+GPU plugin supports the following data types as inference precision of internal primitives:
+
+- Floating-point data types:
+  - f32
+  - f16
+- Quantized data types:
+  - u8
+  - i8
+  - u1
+
+Selected precision of each primitive depends on the operation precision in IR, quantization primitives, and available hardware capabilities.
+u1/u8/i8 data types are used for quantized operations only, i.e. those are not selected automatically for non-quantized operations.
+See [low-precision optimization guide](@ref pot_docs_LowPrecisionOptimizationGuide) for more details on how to get quantized model.
+
+Floating-point precision of a GPU primitive is selected based on operation precision in IR except [compressed f16 IR form](../model_representation.md) which is executed in f16 precision.
+
+> **NOTE**: Harware acceleration for i8/u8 precision may be unavailable on some platforms. In that case model is executed in floating-point precision taken from IR. Hardware support of u8/i8 acceleration can be queried via `ov::device::capabilities` property.
 
-The plugin supports algorithms that fuse several operations into one optimized operation. Refer to the sections below for details.
+[Hello Query Device C++ Sample](../../../samples/cpp/hello_query_device/README.md) can be used to print out supported data types for all detected devices.
 
-> **NOTE**: For operation descriptions, see the [IR Notation Reference](../../ops/opset.md).
+## Supported features
 
-### Fusing Convolution and Simple Layers
+### Multi-device execution
+If a machine has multiple GPUs (for example integrated GPU and discrete Intel GPU), then any supported model can be executed on all GPUs simultaneously.
+This can be achieved by specifying `"MULTI:GPU.1,GPU.0"` as a target device.
 
-Merge of a Convolution layer and any of the simple layers listed below:
-- Activation: ReLU, ELU, Sigmoid, Clamp, and others
-- Depthwise: ScaleShift, PReLU
-- FakeQuantize
+@snippet snippets/gpu/compile_model.cpp compile_model_multi
 
-> **NOTE**: You can have any number and order of simple layers.
+See [Multi-device execution page](../multi_device.md) for more details.
 
-A combination of a Convolution layer and simple layers results in a single fused layer called
-*Convolution*:
-![conv_simple_01]
+### Automatic batching
+GPU plugin is capable of reporting `ov::max_batch_size` and `ov::optimal_batch_size` metrics with respect to the current hardware platform and model,
+thus automatic batching can be applied in cases when `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` is set
+or device is specified as `"BATCH:GPU"`.
```

Comment:
I think it would be better to mention on the default behavior, from benchmark_app and API itself.

Reply:
I don't think that benchmark app should be referred here. From the API perspective I think current statement is enough, so we just say when auto batch is applied for GPU plugin, and all the details on auto batch behavior are expected to be described in dedicated page.